internal/logger/logging_interceptor.go

```diff
@@ -101,11 +102,12 @@ func Interceptor( /*logLevel string, logFormat string, logFile string*/ ) grpc.U
 			logMsg = logger.Error()
 
 			attrs = attrs.Err(err)
-			// Log the body of errors (TODO: do need to scrub requests?)
+			// Log the body of errors (TODO: do we need to scrub requests?)
```

Comment:
I know this code wasn't changed as part of your pR, but isn't the error condition below reversed? It seems to be logging the body only if marshalling the body failed?

Reply:
I wonder if we should actually be logging this... what do you think of removing this code?

Reply:
We're going to put this logging behind a flag, to avoid leaking PII into logs without consideration.  (And fix the condition)